src/utils/componentUtilityFunctions.js

```diff
@@ -2,6 +2,8 @@ import delay from 'lodash/delay';
 import moment from 'moment';
 import { sortStates } from 'carbon-components-react/lib/components/DataTable/state/sorting';
 import fileDownload from 'js-file-download';
+import find from 'lodash/find';
+import some from 'lodash/some';
```

Comment:
Same as before,  this is native in JS https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
in this case I actually want the simpler predicate to make the code more readable